### ISSUE TITLE
Add link to Who's That tool to check past usernames of user

### ISSUE THREAD

**Issue by batpad** [mapbox/osmcha-frontend#111]
From @chtnha : 

It will be very useful in finding🔎 the person who constantly renames their username to continue vandalism if they were caught by community or blocks received by DWG.

I found this tool a very useful in tracking usernames. http://whosthat.osmz.ru/ . I wish this tool be integrated on Details section like HDYC.

From me:

The idea would be to query this API to get a list of past usernames for a user. A user changing their username often could often be a red flag.

To query, make request like http://whosthat.osmz.ru/whosthat.php?action=names&q=Chetan_Gowda and you get a list of past usernames for user Chetan_Gowda. So we probably want something like past_usernames in the UserDetails section. Will see how best to figure this out.

@kepta this is probably not huge priority, but probably a nice-to-have at some point, even as a link from the Users details. Even just a link to Who's That may suffice.

**Comment by kepta** [comment]
This was added

**Comment by rasagy** [comment]
@batpad This has been added as a feature that’ll show past usernames if usernames have been changed (via Whosthat API).

@kepta Did you find an example for this, and can you post it here please? 🙇

**Comment by kepta** [comment]
@rasagy I would search for Chethan to find one.

**Comment by kepta** [comment]
@batpad the  http://whosthat.osmz.ru/whosthat.php?action=names&q=Chetan_Gowda  is served in http and not https. 
This is being blocked by the browser :(.
Do you think we can work around this ?

**Comment by kepta** [comment]
cc @ajithranka do you think we can use proxy to circumvent the limitation of whosthat?

**Comment by rasagy** [comment]
@kepta For now, how about we just show a link to the whosthat page like we are doing for HDYC?

**Comment by rasagy** [comment]
Minor issue: We should either hide the `past username` field when there are no past usernames, or show a more relevant message.

IMO, let’s hide that field unless Whosthat returns a past username, and show it below the current username:

<img width="471" alt="screenshot 2017-07-10_13_13_44 edited" src="https://user-images.githubusercontent.com/1117531/28007748-49576b7e-6572-11e7-98fb-731299e7025f.png">

**Comment by kepta** [comment]
This was fixed.
